src/homekit.cs

```diff
@@ -75,6 +79,14 @@ partial interface HMHomeManagerDelegate {
 
 		[Export ("homeManager:didRemoveHome:"), EventArgs ("HMHomeManager")]
 		void DidRemoveHome (HMHomeManager manager, HMHome home);
+
+		[iOS (13,0), NoWatch, NoTV, NoMac]
+		[Export("homeManager:didReceiveAddAccessoryRequest:")]
+		void DidReceiveAddAccessoryRequest (HMHomeManager manager, HMAddAccessoryRequest request);
```

Comment:
Nit: update to something more like `DidReceiveAddAccessory` or `DidReceiveAdd` since `AddAccessoryRequest` is a suffix that describes one of the parameter values. I'm not sure how familiar with ObjC you are (I'm new to it!) but generally method names tend to contain the names/types of one of their parameters. When translating to C# we usually remove the suffix :)

@dalexsoto can you confirm?